Sources/Contour.Transport.RabbitMq/Internal/DefaultPayloadConverterResolver.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Contour.Serialization;
+
+namespace Contour.Transport.RabbitMq.Internal
+{
+    internal class DefaultPayloadConverterResolver : IPayloadConverterResolver
+    {
+        private readonly string firstContentType;
+        private readonly IDictionary<string, IPayloadConverter> converters;
+
+        public DefaultPayloadConverterResolver(IReadOnlyCollection<IPayloadConverter> converters)
+        {
+            this.converters = converters.ToDictionary(c => c.ContentType, c => c);

```

Comment:
I think it should not contain duplicate content types. We can override content type in the collection by a duplicate or raise an exception. What is the best choice?